primitives/arithmetic/src/biguint.rs

```diff
@@ -734,6 +739,7 @@ pub mod tests {
 	fn div_unit_works() {
 		let a = BigUint { digits: vec![100] };
 		let b = BigUint { digits: vec![1, 100] };
+		let c = BigUint { digits: vec![14, 28, 100] };
```

Comment:
Nothing against new tests, but wondering what are your intending to test here?

Reply:
obvious 14/2=7  28/2=14 
This is more intuitive and better understood